app/executor.php

```diff
@@ -188,6 +192,19 @@ function getStorageDevice($root): Device
         $endTime = 0;
         $orchestration = $orchestrationPool->get();
 
+        $secret = \bin2hex(\random_bytes(16));
+
+        if (!$remove) {
+            $activeRuntimes->set($runtimeId, [
+                'id' => $containerId,
+                'name' => $runtimeId,
+                'created' => $startTime,
+                'updated' => $endTime,
+                'status' => 'pending',
```

Comment:
This might be a bug. Should we set here to pending, or running?

Reply:
We set it to pending since we haven't launched the runtime yet, we change it to running later on
https://github.com/appwrite/appwrite/blob/515c8c2e3212dd6e93ecccfed4a297a38cd6f035/app/executor.php#L332-L341

Reply:
This does remind me to handle errors though...